.github/workflows/ci.yml

```diff
@@ -9,19 +9,25 @@ on:
 jobs:
 
   build:
-    name: Build
+    name: Components tests
     runs-on: Ubuntu-20.04
     strategy:
       fail-fast: false
       matrix:
         php: ['7.2', '7.3', '7.4', '8.0']
+        strategy: ['default']
+        include:
+        # Minimum supported dependencies with the oldest PHP version
+        - php: '7.2'
+          strategy: 'lowest'
 
     steps:
       - name: Set up PHP
         uses: shivammathur/setup-php@2.7.0
         with:
           php-version: ${{ matrix.php }}
           coverage: none
+          tools: ${{ matrix.strategy == 'lowest' && 'flex' || '' }}
```

Comment:
We can always add flex as a tool.

Reply:
no strong opinion about this. But I feel it's no more necessary with `composer 2` (unless combined with SYMFONY_REQUIRE).

I changed it to always add flex.
BUT I also change the `SYMFONY_REQUIRE` to use `>=` constraint and avoid that someday we forgot to include latest releases

Reply:
I wanted to always include flex because the config file would be simpler. I agree, there is no real benefit on the actual execution.